docs/general_usage.md

```diff
@@ -10,6 +10,8 @@ Alternatively, you may switch which profile is active using the `twilio profiles
 
 To take advantage of Twilio's [Global Infrastructure], you can specify the target Region for the profile using the `--region` flag (e.g., `twilio login --region au1`).
 
+_NOTE: The Twilio CLI does not currently support logging in with test credentials._
```

Comment:
Instead of `The Twilio CLI` we've been trying to stay consistent and just call it `twilio-cli`.